src/components/PokeCard/PokeCard.js

```diff
@@ -0,0 +1,56 @@
+import axios from "axios";
+import React, {useEffect, useState} from "react";
+import './PokeCard.css'
+
+function PokeCard({ url }) {
+    const [pokemonData, setPokemonData] = useState([]);
```

Comment:
Zodra de setter aangeroepen is, dan is `pokemon` niet meer van het type array. Ik zou daarom de default waarde `undefined` maken
```suggestion
    const [pokemonData, setPokemonData] = useState();
```

Reply:
Dan krijg ik deze foutmelding: Cannot read properties of undefined (reading 'previous') Dus ik laat de array staan